perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -3837,14 +3835,10 @@ private void tryToAutoApproveApplication(PerunSession sess, Application app) thr
 
 			}
 		} catch (Exception ex) {
-
-			ArrayList<Exception> list = new ArrayList<>();
-			list.add(ex);
-			getMailManager().sendMessage(app, MailType.APP_ERROR_VO_ADMIN, null, list);
-
+			setAutoApproveErrorToApplication(app, ex.getMessage());
+			getMailManager().sendMessage(app, MailType.APP_ERROR_VO_ADMIN, ex.getMessage(), Arrays.asList(ex));
```

Comment:
I'm not sure, that passing `ex.getMessage()` as a `reason` param to the notification is useful, since it was originally used only in "rejection" notification, but I checked and it can be now generally used in all notifications (replaces tag `{customMessage}`. For error messages we used exception objects with tag `{errors}`.

So you can probably use it to generally improve the notification style/content, but make sure to think about it from the perspective of hints in GUI, documentation and mention it in the release notes for new version.